dist/Devel-PPPort/parts/inc/inctools

```diff
@@ -101,6 +101,22 @@ sub dictionary_order($$)    # Sort caselessly, ignoring punct
         ($valid_a, $valid_b) = @_;
     }
 
+    if ("$]" >= 7.0000) {
+        return _dictionary_sort_internals_7(
+            $lc_a, $lc_b, $squeezed_a, $squeezed_b, $valid_a, $valid_b
+        );
+    }
+    else {
+        return _dictionary_sort_internals_5(
+            $lc_a, $lc_b, $squeezed_a, $squeezed_b, $valid_a, $valid_b
+        );
+    }
+}
+
+sub _dictionary_sort_internals_7 {
+    my ($lc_a, $lc_b, $squeezed_a, $squeezed_b, $valid_a, $valid_b) = @_;
+
+    no warnings 'uninitialized';
```

Comment:
alternatively we could have avoided the creation of another function by using something like this
`no if "$]" >= 7, warnings 'uninitialized';`

Reply:
Yeah, I was grasping for that earlier today but couldn't get it right.

Fixed in 187aa25307.